src/settings/applicationStartupActions.ts

```diff
@@ -0,0 +1,14 @@
+import { ApplicationStartup } from '../types/applicationStartup';
+import { setSetting } from './globalSettingsActions';
+
+const onboarding: ApplicationStartup = {
```

Comment:
Comments will be added

Reply:
onboarding er vel strengt tatt ikke en module men, jeg ser hva du mener. Bør det være en del av rammeverket?

Reply:
Det bør være endel av de 2 onboarding filene, som sannsynligvis havner i rammeverket ja :)

Reply:
skal jeg ta det nå på denne pr syns du?

Reply:
Det bør tas vekk herifra ja :)